pages/index.js

```diff
@@ -1,63 +1,165 @@
-import { exercises as initialExercises } from "@/lib/exercises";
-import { useState } from "react";
 import {
-  StyledFlexWrapper,
-  ExerciseList,
-  StyledFilterButton,
-  StyledFilterWrapper,
-} from "/styledComponents.js";
-import Exercise from "../components/Exercise/Exercise";
-import FilterSection from "@/components/FilterSection/FilterSection";
+  motivationalQuotesBad,
+  motivationalQuotesGood,
+  motivationalQuotesOkay,
+} from "@/lib/quotes";
+import { useState } from "react";
+import { useEffect } from "react";
+import Image from "next/image";
+import styled from "styled-components";
 
 export default function HomePage() {
-  const [exercises, setExercises] = useState(initialExercises);
-  const [filters, setFilters] = useState([]);
-  const [filterMode, setFilterMode] = useState(false);
-
-  function filterExercises(exercises, filters) {
-    const filteredArray = exercises.filter((exercise) => {
-      return filters.every((filter) => exercise.muscleGroups.includes(filter));
-    });
-    return filteredArray;
-  }
+  const quotes = [
+    ...motivationalQuotesBad,
+    ...motivationalQuotesGood,
+    ...motivationalQuotesOkay,
+  ];
+
+  const [currentQuote, setCurrentQuote] = useState({
+    quote: "Loading...",
+    author: "",
+  });
+  const [moodMode, setMoodMode] = useState(null);
 
-  function onAddFilter(newFilter) {
-    setFilters([...filters, newFilter]);
+  useEffect(() => {
+    setCurrentQuote(quotes[getRandomIndex(quotes.length)]);
+  }, []);
```

Comment:
It seems like you do not need this `useEffect`. The user will either see the form to enter a mood or a quote, once they have chosen a mood. 

Therefore you also won't need the constant `quotes`, as you will only use the `motivationalQuotesBad`, `motivationalQuotesOkay` and `motivationalQuotesGood` to find a quote fitting to the chosen mood.